.circleci/config.yml

```diff
@@ -510,6 +510,51 @@ jobs:
           command: |
             docker push flowminder/<<parameters.component>>:$CIRCLE_SHA1
 
+  flowetl_build_and_test:
+    docker:
+      - image: circleci/python:3.7
+    working_directory: /home/circleci/project/
+    steps:
+      - checkout:
+          path: /home/circleci/project/
+      - setup_remote_docker
+      - run:
+          name: Log in to docker cloud
+          command: docker login --username $DOCKER_CLOUD_USER --password $DOCKER_CLOUD_PASSWORD
+      - run:
```

Comment:
Can we _split_ this, rather than just shove it all into a different job?

So, this job (sans the build), downstream of an instance of the build docker image job, and then pull the image built in that.